Source/Core/DolphinQt2/QtUtils/ListTabWidget.cpp

```diff
@@ -0,0 +1,78 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "DolphinQt2/QtUtils/ListTabWidget.h"
+
+#include <QHBoxLayout>
+#include <QListWidget>
+#include <QScrollBar>
+#include <QStackedWidget>
+
+class FixedWidthAlwaysOneSelectedListWidget : public QListWidget
```

Comment:
Thats... quite a name. How about `CategoryListWidget`? You can add a comment describing what this class actually does, but IMO long class names are annoying.